### DISCUSSION TITLE
Translate Plausible

### DISCUSSION THREAD

**Discussion by ConsultorML** [plausible/analytics#1478] [Anything Else]
Hi, first of all, excellent software. Simple and direct for what most people actually need. I want to contribute translating to spanish. How I can do it?

**Comment by metmarkosaric** [comment]
thanks @ConsultorML! Plausible is in English only at the moment. hopefully in the future we'll make it multilingual and then people will be able to contribute with their own languages. you can follow the progress on this in this thread: #354
